Userland/Services/WindowServer/WindowServer.ipc

```diff
@@ -136,6 +136,7 @@ endpoint WindowServer
 
     get_screen_bitmap(Optional<Gfx::IntRect> rect, Optional<u32> screen_index) => (Gfx::ShareableBitmap bitmap)
     get_screen_bitmap_around_cursor(Gfx::IntSize size) => (Gfx::ShareableBitmap bitmap)
+    get_color_under_cursor() => (Gfx::Color color)
```

Comment:
This may be a stupid question, but why not use `get_screen_bitmap_around_cursor({1, 1})` instead?

Reply:
Andreas pointed out that it's not ideal allocating a Bitmap and sending it through IPC as a shared buffer just to get one pixel, which is why I added this ability. If you look at the original PR linked, that's what I was doing!

Reply:
Sure, allocating a 1024x768 pixel bitmap hast lots of disadvantages. But which ones still apply to a 1x1 bitmap?

Oh, I see, that's still what Andreas argued against. Alright, I don't see it like that, but it answers my question. Thanks!

Reply:
I think it's got more to do with having a Bitmap do a dynamic allocation, which again gets complicated when doing IPC since now we need to make it into a shared buffer, and then it's copied again on the client end. It's not really a question of it taking too much memory but that there's no need to go through all that trouble when we can directly just ask for the color. The API was trivial enough to add, just querying the bitmap.

(or, that's my take on it, at least. You can watch the first few minutes of the office hours from last week where he talks about it)